.devcontainer/Dockerfile

```diff
@@ -1,3 +1,7 @@
-FROM ubuntu:latest
+FROM debian:latest
 
-RUN apt-get update && apt-get install -y p7zip-full binutils-mips-linux-gnu build-essential pkgconf python3 git zip wget curl libsdl2-dev emscripten
+RUN apt-get update && apt-get install -y wget curl gpg
+RUN mkdir -p /etc/apt/keyrings
+RUN curl -fsSL https://ethanthesleepy.one/deb-repo/ethanthesleepyone.pgp.key | gpg --dearmor -o /etc/apt/keyrings/ethanthesleepyone.gpg
+RUN echo "deb [signed-by=/etc/apt/keyrings/ethanthesleepyone.gpg] https://ethanthesleepy.one/deb-repo/ nodistro main" | tee /etc/apt/sources.list.d/ethanthesleepyone.list
+RUN apt-get update && apt-get install -y p7zip-full binutils-mips-linux-gnu build-essential pkgconf python3 git zip libsdl2-dev emscripten jq
```

Comment:
Could we add to the readme instructions for people who dont like to use docker for little things like this.

(I can say I am one of those people, I dont even use a "real" IDE. Theyre all bloated and full of spyware nowadays)

Reply:
Absolutely - I hadn't done that part yet. Was waiting for your review :)

Reply:
It's already in place :) Look under the ``# Set up your build environment`` and then ``## Local``.

I did update the Readme to say that you may need to run the script as sudo.